model/src/pyrenew/process/rtrandomwalk.py

```diff
@@ -24,34 +24,29 @@ class RtRandomWalkProcess(RandomVariable):
 
     def __init__(
         self,
-        Rt0_dist: dist.Distribution = dist.TruncatedNormal(
-            loc=1.2, scale=0.2, low=0
-        ),
+        Rt0_dist: dist.Distribution,
+        Rt_rw_dist: dist.Distribution,
```

Comment:
I think these should both be random variables.

Reply:
I'm open to this being a separate issue.

Reply:
100% Agree on this one